DirectXTex/DirectXTexImage.cpp

```diff
@@ -11,14 +11,8 @@
 
 #include "DirectXTexP.h"
 
-namespace DirectX
-{
-    extern bool _CalculateMipLevels(_In_ size_t width, _In_ size_t height, _Inout_ size_t& mipLevels) noexcept;
```

Comment:
There were a few places I had done module-to-module links like this. I moved them all to DirectXTexP.h instead.